packages/frontend/src/components/matchcard.tsx

```diff
@@ -0,0 +1,67 @@
+import type { Sport } from "@esp-group-one/types";
+import { Tag } from "./tags";
+import { Profile } from "./profile";
+
+interface MatchCard {
+  sportsTag: Sport;
+  profilePic: string;
+  name: string;
+  date: string;
+  startTime: string;
+  endTime: string;
+  link: string;
+}
+
+export function Match({
```

Comment:
Could we change the name to `MatchCard` so its distinguishable to our other type `Match`